packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -597,8 +605,8 @@ export class GarbageCollector implements IGarbageCollector {
             return baseGCDetailsMap;
         });
 
-        // Initialize the base state. The base GC data is used to detect and log when inactive / deleted objects are
-        // used in the container.
+        // Initialize the base state that is used to detect when inactive objects are used. It is explicitly initialized
+        // for non-summarizer clients as they don't run GC. Summarizer clients do this when GC runs first time.
```

Comment:
I don't understand how this comment relates to the code.  It sounds like there would be some "explicit initialization" happening in an `else` block or something.

Reply:
Good catch. I had added a check for summarizer client here but removed it later. The comment is left over from that.